extensions/wikia/WikiaRSS/WikiaRssExternalController.class.php

```diff
@@ -127,6 +133,12 @@ private function getFullItemList($items, $options) {
 			if( $item['description'] ) {
 				$text = trim(mb_convert_encoding($item['description'],$outputEncoding,$charset));
 				$text = str_replace( array("\r", "\n", "\t", '<br>'), ' ', $text );
+				$text = strip_tags( $text, '<img><a><br>' );
```

Comment:
`removeHTMLtags()` is more complex method. By defaults it strips tags that are not whitelisted. The same rule is applied to attributes. Other tags are kept in place. Unfortunately this method can't handle more complex HTML (or not fully valid HTML) that's why we limit allowed tags using `strip_tags()`.